security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,22 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.11.4"]
+== Release notes for {cert-manager-operator} 1.11.4
+
+Issued: 2023-07-26
+
+The following advisory is available for the {cert-manager-operator} 1.11.4:
+
+* link:https://access.redhat.com/errata/RHEA-2023:4081[RHEA-2023:4081]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11].
+
+[id="cert-manager-operator-1.11.4-bug-fixes"]
+=== Bug fixes
+
+*
```

Comment:
You can add: https://issues.redhat.com/browse/OCPBUGS-16393 to this list.

Reply:
Addressed